pombola/writeinpublic/views.py

```diff
@@ -218,11 +212,32 @@ def get_template_names(self):
         return [self.adapter.get_templates()[self.steps.current]]
 
     def get_context_data(self, form, **kwargs):
+        step = kwargs.get('step')
         context = super(WriteInPublicNewMessage, self).get_context_data(form=form, **kwargs)
         context['message'] = self.get_cleaned_data_for_step('draft')
         recipients = self.get_cleaned_data_for_step('recipients')
+
         if recipients is not None:
-            context['persons'] = recipients.get('persons')
+            persons = recipients.get('persons')
+            if step == 'draft':
+                # On the draft step, check if all recipients are contactable
+                # in WriteInPublic
+                not_contactable = []
+                contactable = []
+                for person in persons:
+                    try:
+                        if not self.client.get_person_is_contactable(person):
+                            not_contactable.append(person)
+                        else:
+                            contactable.append(person)
+                    except Exception:
+                        not_contactable.append(person)
```

Comment:
always log exceptions, ideally in a way sentry will find them. Looks like logger.exception will do the right thing
https://docs.sentry.io/platforms/python/logging/
This is for things like http exceptions, right? we want the user to proceed but we want to know the system is funky